src/components/DownloadImageDialog/ApplicationInstructions.tsx

```diff
@@ -190,7 +208,30 @@ const InstructionsItem = ({ node, index }: InstructionsItemProps) => {
 						return <p />;
 					},
 					p: ({ children }) => (
-						<p style={{ marginTop: 0, marginBottom: 0 }}>{children}</p>
+						<p style={{ marginTop: 0, marginBottom: 0 }}>
+							{children}
+							{secureboot && index === 7 && (
```

Comment:
> Isn't there a better way to detect this? Like find the instruction step that includes x/y/z?
We should for sure also have a `TODO find a way to do this via contracts` or something.